src/mappings/PoolRewards.ts

```diff
@@ -112,7 +110,7 @@ async function handleRelaychainPooledStakingSlash(
         if (member.poolId.toNumber() === poolId) {
             memberPoints = memberPointsCounter(member)
             if (memberPoints != BigInt(0)) {
-                const personalSlash = (slash / poolPoints) * memberPoints
```

Comment:
Better to use BigInt functions for arithmetics like this https://github.com/novasamatech/subquery-staking/blob/master/src/mappings/rewards/CollatorStakingRewardCalculator.ts#L34